### PR TITLE
Update hybrid_identity.py

### PR DESCRIPTION
**Pull request by symonds**
Icehouse compatible hybrid_identity.py

### CODE REVIEW

**Comment by alex-leonhardt** [comment]
:+1:

**Comment by vuntz** [comment]
@mapleoin what do you think?

File: hybrid_identity.py [hunk 1]
```diff
@@ -85,11 +87,11 @@ def _get_user(self, session, user_id):
             # then try LDAP
             return self.user.get(user_id)
         else:
-            return user_ref.to_dict()
```
Comment (iartarisi) [inline hybrid_identity.py:88]: I don't understand why you changed this. You're just redoing the same call as in line 83 above.

File: hybrid_identity.py [hunk 2]
```diff
@@ -20,15 +20,17 @@
 from keystone.common import utils
 from keystone import exception
 from keystone import identity
+from keystone.common import driver_hints
```
Comment (iartarisi) [inline hybrid_identity.py:23]: this import is unused

**Comment by iartarisi** [comment]
Thanks for the pull request, @msymonds! This looks good. I have a couple of suggestions inline. 

Also, could you please reopen this against the newly-created `devel` branch? It wouldn't make sense to merge it into the stable/havana branch.

**Comment by symonds** [comment]
Thanks for the comments @mapleoin you are correct on both comments. I will close this request and reopen against the devel branch.

**Comment by iartarisi** [comment]
This discussion continued in #7.

**Pull request closed without merging**